index.html

```diff
@@ -1,17 +1,82 @@
     />
     <meta name="author" content="Daniel Rodriguez" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Hello Microverse!</title>
+    <title>Daniel Rodriguez Developer Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:wght@100;200;300;400&family=Inter:wght@100;200;300;400;500;600;700&family=Poppins:wght@200;300;400&family=Roboto:wght@500;700&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
-    <h1>Hello Microverse!</h1>
+    <div class="container">
+      <header class="header">
+        <a class="header__logo" href="#">My Logo</a>
+        <nav class="header__nav">
+          <li class="nav__item--icon">
+            <svg class="icon icon--m">
+              <use xlink:href="./images/sprite.svg#menu"></use>
+            </svg>  
+          </li>
+        </nav>
+      </header>
+      <main class="hero">
+        <h1 class="hero__heading">
+          <span>Hey There.</span> <span>I'm Daniel.</span>
+        </h1>
+        <h3 class="hero__subheading">I'm a Software Developer</h3>
+        <p class="hero__blurb">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <ul class="hero__socials">
+          <li class="socials__icon">
+            <a href="" class="socials__github">
+              <svg class="icon">
+                <use xlink:href="./images/sprite.svg#github"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="socials__icon">
+            <a href="" class="socials__linkedin">
+              <svg class="icon">
+                <use xlink:href="./images/sprite.svg#linkedin"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="socials__icon">
+            <a href="" class="socials__angellist">
+              <svg class="icon">
+                <use xlink:href="./images/sprite.svg#angellist"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="socials__icon">
+            <a href="#about" class="socials__twitter">
+              <svg class="icon">
+                <use xlink:href="./images/sprite.svg#twitter"></use>
+              </svg>
+            </a>
+          </li>
+          <li class="socials__icon">
+            <a href="" class="socials__medium">
+              <svg class="icon">
+                <use xlink:href="./images/sprite.svg#medium"></use>
+              </svg>
+            </a>
+          </li>
+        </ul>
+      </main>
```

Comment:
Kindly use a `section` tag for this [part of your project as the container of a section is a `section` tag, you can check this link for more details [url](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

Reply:
@usorfaitheloho  thank you for your suggested change. I changed the hero section tag from a main tag to a section tag.